lib/aquilon/worker/templates/base.py

```diff
@@ -629,18 +629,20 @@ def transaction(self, verbose=False):
 
 
 def add_location_info(lines, dblocation, prefix=""):
```

Comment:
In general we agree - it is a good fix - but we have to make huge template change change to allow this to go out cause we have some sysloc template code that would start failing if we just roll this out... I suggest to create a separate PR for this if we want to release the other changes sooner :)

Reply:
I'll look it if makes sense but will probably not do anything before end of August anyway... What's your timeframe?